go/enclave/components/batch_producer.go

```diff
@@ -62,7 +62,7 @@ func (bp *batchProducerImpl) ComputeBatch(context *BatchExecutionContext) (*Comp
 	}
 
 	// Create a new batch based on the fromBlock of inclusion of the previous, including all new transactions
-	batch := core.DeterministicEmptyBatch(context.Creator, parent, block, context.Randomness, context.AtTime)
+	batch := core.DeterministicEmptyBatch(parent, block, context.Randomness, context.AtTime)
```

Comment:
Why are we removing the sequencer address per batch?

Reply:
the signature already contains that information

Reply:
we're not even using that.
We have the approved sequencer stored in the DB and check the signature against that key.